### PR TITLE
Refresh button

### PR DESCRIPTION
**Pull request by Khalidxx**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | 
| Patch: Bug Fix?          |
| Major: Breaking Change?  |
| Minor: New Feature?      | Yes
| Tests Added + Pass?      | Yes
| Does not reduce coverage?| Yes
| Any Dependency Changes?  | No

Added a refresh button to the trading page to call the `proposal_open_contract` api and repopulate the trading table

### CODE REVIEW

File: src/botPage/view/TradeInfoPanel/TradeTable.js [hunk 1]
```diff
@@ -126,6 +139,38 @@ export default class TradeTable extends Component {
         }
         return this.state[accountID];
     }
+    updateTable(info) {
+        if (!info) {
+            return;
+        }
+        const buyDate = new Date(info.date_start * 1000);
+        const timestamp = `${buyDate.toISOString().split('T')[0]} ${buyDate.toTimeString().slice(0, 8)} ${
```
Comment (aaron-binary) [inline src/botPage/view/TradeInfoPanel/TradeTable.js]: Since we're using this in both the `bot.contract` event (see `componentDidMount()`) and here, we could create a common function for parsing the timestamp.

File: static/image/refresh-disabled.svg [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="24" height="24">
```
Comment (aaron-binary) [inline static/image/refresh-disabled.svg]: You can run `svgo` to reduce the size of this file, see https://github.com/svg/svgo

Comment (aaron-binary) [inline static/image/refresh-disabled.svg]: You can create a disabled version of the `refresh.svg` file by creating a copy of it and adding a `fill` attribute with value `#dedede` to the first `path` element in this new copy.

File: static/css/_panel.scss [hunk 3]
```diff
@@ -207,6 +207,10 @@ $disabled-color: #F2F2F2;
             position: absolute;
             right: 10px;
         }
+        #refreshButton {
+            position: absolute;
+            right: 80px;
```
Comment (aaron-binary) [inline static/css/_panel.scss]: Could you discuss with Lucas the spacing required between these icons?

File: gulpfile.babel.js [hunk 4]
```diff
@@ -24,7 +24,7 @@ gulp.task(
     gulp.series(done => {
         gulp.src('www/index.html').pipe(
             open({
-                uri: 'http://localhost:8080/',
+                uri: 'http://localhost:80/',
```
Comment (aaron-binary) [inline gulpfile.babel.js]: Please revert this change.

File: static/image/refresh-disabled.svg (hunk 2 above)
Comment (cakasuma) [inline static/image/refresh-disabled.svg]: `svgo` ?

File: gulpfile.babel.js [hunk 5]
```diff
@@ -12,7 +12,7 @@ gulp.task(
     gulp.series(done => {
         connect.server({
             root      : 'www',
-            port      : 8080,
+            port      : 80,
```
[the remark this answers is not in the document]
Comment (Khalidxx) [reply gulpfile.babel.js]: Yeah, it's a hack for redirecting to localhost after loging in (issue #16014)

File: static/css/_fontello.scss [hunk 6]
```diff
@@ -89,6 +89,15 @@
         background: url("../image/clear-disabled.svg") no-repeat center;
     }
 }
+.icon-refresh {
```
[the remark this answers is not in the document]
Comment (Khalidxx) [reply static/css/_fontello.scss]: Yeah, this seems more appropriate. But I was following the other classes (`icon-hide-header` and `icon-clear`) just for consistency. 
@aaron-binary ?

File: static/image/refresh-disabled.svg (hunk 2 above)
Comment (Khalidxx) [reply static/image/refresh-disabled.svg]: And I used photoshop to generate an entirely new file.. 😩

File: static/css/_fontello.scss (hunk 6 above)
[the remark this answers is not in the document]
Comment (aaron-binary) [reply static/css/_fontello.scss]: Yes, @cakasuma's suggestion looks much better, you can update the other classes as well.
